client/src/components/App/index.js

```diff
@@ -97,7 +99,7 @@ export default class App extends React.Component {
 
           <main className="container">
             {Routes.includes(pathname) ? (
-              loading ? (
+              loading === false ? (
```

Comment:
keep it as before , this is for debugging for you until we finished login 
```suggestion
              loading ? (
```